packages/api/src/car.js

```diff
@@ -6,6 +6,7 @@ import { toString, equals } from 'uint8arrays'
 import { LinkIndexer } from 'linkdex'
```

Comment:
Not sure where these changes are coming from? I wonder if this is coming from a bad merge.
Pretty sure all changes in API package needs to be reverted to whatever we have in main

Reply:
Reverted changest to the API